js/main.js

```diff
@@ -1592,6 +1591,20 @@ function calculateGridLoad() {
     return gridLoad;
 }
 
+function daysToCycles(currentDays, baseDays) {
+    const decimalPlaces = 0;
+    const incrementPerDay = 1;
+
+    const calculatedStardate = baseDays + incrementPerDay * currentDays;
+    const roundedStardate = calculatedStardate.toFixed(decimalPlaces);
+  
+    return roundedStardate;
+}
+
+function formatValueWithCommas(x) {
+    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
 function daysToYears(days) {
```

Comment:
Die methode braucht es dann wohl nicht mehr

Reply:
wird in verschiedenen 'Requirements' noch verwendet.

Reply:
scheinbar sind die aus irgendeinem grund in "jahren" definiert?